docs/design/architecture/guest-assets.md

```diff
@@ -135,7 +135,6 @@ See also the [process overview](README.md#process-overview).
 
 | Image type | Default distro | Init daemon | Reason | Notes |
 |-|-|-|-|-|
-| [image](background.md#root-filesystem-image) | [Clear Linux](https://clearlinux.org) (for x86_64 systems)| systemd | Minimal and highly optimized | systemd offers flexibility |
```

Comment:
should we keep the line and just `s/Clear Linux/Ubuntu/` ?
And maybe change the Reason to: `Fully tested in our CI`